eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Serialisation/EraIndepGenerators.hs

```diff
@@ -95,237 +86,21 @@ instance Mock c => Arbitrary (BHeader c) where
         bodySize
         bodyHash
 
-instance Crypto c => Arbitrary (TP.HashHeader c) where
+instance Crypto c => Arbitrary (TP.HashHeader c) where -- NOTE: this cannot be moved to cardano-ledger-shelley because it is from cardano-protocol-tpraos
```

Comment:
I think this should work:

Move it into `cardano-protocol-tpraos:testlib` and make it depend on `cardano-ledger-shelley:testlib`.

Here is an example dependency graph:

```
cardano-ledger-shelley-test --> cardano-protocol-tpraos:testlib
                                                   \      \
                                                      `----`> cardano-ledger-shelley:{cardano-ledger-shelley, testlib}
```

Reply:
We decided to address this in a follow up PR, as this one is getting big.

Reply:
I know, but the comment is wrong and needs to be removed

Reply:
Agree! Changed to `TODO: Move to TPraos or Shelley`.